src/v3/core/graph.test.js

```diff
@@ -37,5 +37,60 @@ describe("core/graph", () => {
       // $ExpectFlowError
       expect(() => x.measureSpectacularity()).toThrow();
     });
+    function graphRejectsNulls(f) {
+      [null, undefined].forEach((bad) => {
+        it(`${f.name} errors on ${String(bad)}`, () => {
+          // $ExpectFlowError
+          expect(() => f.call(new Graph(), bad)).toThrow(String(bad));
+        });
+      });
+    }
+    describe("node methods", () => {
+      const p = Graph.prototype;
+      const nodeMethods = [p.addNode, p.removeNode, p.hasNode];
+
+      describe("error on", () => {
+        function rejectsEdge(f) {
+          it(`${f.name} rejects EdgeAddress`, () => {
+            const e = Address.edgeAddress(["foo"]);
+            // $ExpectFlowError
+            expect(() => f.call(new Graph(), e)).toThrow("EdgeAddress");
```

Comment:
Can the expected message be `expected NodeAddress` or `got EdgeAddress`?
As it stands, if we had copy-pasted the error message from a node method
to an edge method, the test wouldn’t catch this.